app/views/declaration/summary/amend_summary_page.scala.html

```diff
@@ -35,7 +37,8 @@
         items_section: items_section,
         warehouse_section: warehouse_section,
         transport_section: transport_section,
-        linkButton: linkButton
+        label: label,
+        link: link
```

Comment:
I don't understand this change. Why the `linkButton` component is not sufficient? It's purpose was to have a link that looks like a button.

Reply:
The AC on the ticket is "Change the 'Continue' buttons to a link on both pages", so we are replacing the button looking link with a link looking link :wink:

Reply:
Okay, didn't think about that. No probs for me then.